tests/PHPUnit/Command/SynoliaSchedulerRunCommandTest.php

```diff
@@ -78,6 +80,52 @@ public function testExecuteWithANewCommandThatItMarkedAsExecuteNow(): void
         self::assertStringNotContainsString('Nothing to do', $commandTester->getDisplay());
     }
 
+    /**
+     * @group toto
```

Comment:
What is this group ?

Reply:
I see it, and i'll remove it, thanks for report.

Reply:
Oh sorry I forgot to remove it. It just for me when I debug this test. 
Thanks for repport @Jibbarth